### ISSUE TITLE
1.14.3 maven upload breaks the jar!?

### ISSUE THREAD

**Issue by Aang23** [Aang23/UndergroundBiomesConstructs#114]

**Comment by lofcz** [comment]
Yeah, can confirm. Not sure why yet but I should be able to provide a PR later if you have other priorities now. Thanks for your amazing work @Aang23 !

**Comment by Aang23** [comment]
I got another kind of gamebreaking bug right now, so it would be awesome if you managed to fix that !
What I could understand by taking a quick look at the generated jar is that it's doesn't seem to be remapped correctly.

**Comment by lofcz** [comment]
There's some confusion going on about 1.14.2 & 1.14.3, slight adjustments need to take place in order for the jar to work again.
